### DISCUSSION TITLE
Search issue

### DISCUSSION THREAD

**Discussion by AntonNos** [pradyunsg/furo#300] [Potential Issue] [answered]
Hello!
I'm using Furo template to build docs and search is not working on single html pages.
https://docs.mirantis.com/mos/latest/single/
On multi page website all works good (https://docs.mirantis.com/mos/latest/overview.html).
Help me please with this issue.
Thanks in advance!

**Comment by pradyunsg** [comment, marked as answer]
Heyo! This looks like a significantly customised site. Neat.

That said, I don't really have any personal interest in the singlehtml mode of Sphinx, so I won't be investing this further. My guess is that the search page/url is different in singlehtml mode.

I'd be happy to accept PRs fixing this.
